backend/src/router_test.go

```diff
@@ -450,11 +450,23 @@ func (suite *RouterSuite) TestProfile() {
 		suite.JSONEq(`{
 			"user": `+stringify(User{Username: "Username"})+`
 		}`, w.Body.String())
+		
+		w = PATCH("/profile", gin.H{"firstname": "Elk","lastname": "Cloner",}, cookies...)
+		suite.Equal(200, w.Code)
```

Comment:
Mentioned in Discord but we also need to check that the user profile was updated correctly. I think just getting `/profile` is fine for that.

Reply:
Added it